modules/maven3/manifests/init.pp

```diff
@@ -5,37 +5,45 @@
 # @example
 #  include maven3
 #
-#  class { 'maven3': 
+#  class { 'maven3':
 #    version => '3.5.3',
 #  }
 
 class maven3 (
-  $version = '3.5.3',
+  $maven_version = '3.5.3',
```

Comment:
Why version number was provided in code? As I remember the deal was to using parameters for providing version numbers

Reply:
I used it as a default value. You can overwrite it when class is called.

Reply:
lets stick to "latest"  or "installed" as default value for version of any package.